docs/operator-manual/rbac.md

```diff
@@ -275,96 +381,21 @@ data:
     g, my-org:team-qa, role:tester
 ```
 
-## Anonymous Access
-
-The anonymous access to Argo CD can be enabled using `users.anonymous.enabled` field in `argocd-cm` (see [argocd-cm.yaml](argocd-cm.yaml)).
-The anonymous users get default role permissions specified by `policy.default` in `argocd-rbac-cm.yaml`. For read-only access you'll want `policy.default: role:readonly` as above
-
 ## Validating and testing your RBAC policies
 
 If you want to ensure that your RBAC policies are working as expected, you can
-use the `argocd admin settings rbac` command to validate them. This tool allows you to
-test whether a certain role or subject can perform the requested action with a
-policy that's not live yet in the system, i.e. from a local file or config map.
-Additionally, it can be used against the live policy in the cluster your Argo
-CD is running in.
-
-To check whether your new policy is valid and understood by Argo CD's RBAC
-implementation, you can use the `argocd admin settings rbac validate` command.
+use the [`argocd admin settings rbac` command](../user-guide/commands/argocd_admin_settings_rbac.md) to validate them.
+This tool allows you to test whether a certain role or subject can perform the requested action with a policy
+that's not live yet in the system, i.e. from a local file or config map.
+Additionally, it can be used against the live policy in the cluster your Argo CD is running in.
 
 ### Validating a policy
 
-To validate a policy stored in a local text file:
```

Comment:
I would prefer to keep the examples in the doc.

Reply:
duplicated from the CLI command examples and replaced by the link to the CLI command

Reply:
Ok.. in this case, please clarify in the doc that they can refer to the CLI command doc to see usage examples.